src/controllers/UserController.js

```diff
@@ -0,0 +1,43 @@
+const User = require('../models').User;
+const { getHashedPassword } = require('../helpers/helpers');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+module.exports = {
+  async register(req, res) {
+    try {
+      const { username, password } = req.body;
+      const alreadyExistsUser = await User.findOne({ where: { username } });
+      if (alreadyExistsUser) {
+        return res.status(400).send({ error: 'User with this username already exists' });
+      }
+      const hashedPassword = await getHashedPassword(password);
```

Comment:
Перенеси хеширование в модель https://sequelize.org/master/manual/getters-setters-virtuals.html
```suggestion
      const hashedPassword = await getHashedPassword(password);
```

Reply:
Перенес